### ISSUE TITLE
Add tests for Loggly integration

### ISSUE THREAD

**Issue by eho-tacc** [TACC-Cloud/python-reactors#33]
We need to cover the use case where a base image with loggly integration receives a 200 response from Loggly. 

## Solution

Run pytests again, but within base image https://github.com/TACC-Cloud/python-reactors/blob/619c391e18667f6b8e5240cc1d9ad3b47aaaa31e/.github/workflows/docker-publish.yml#L94-L94

**Comment by eho-tacc** [comment]
@shwetagopaul92 any updates on Loggly manual local testing?

**Comment by shwetagopaul92** [comment]
@eho-tacc Yes, the manual testing worked fine. I have attached a screenshot to show the logs on Loggly Interface.  I am currently working from the branch feat-loggly-integration to add tests with the customer_token set in Github Secrets.  

![Screen Shot 2021-07-28 at 5 35 08 PM](https://user-images.githubusercontent.com/20664289/129266281-8f97f016-c901-4062-b0ec-5eb02f2ff899.png)

**Comment by eho-tacc** [comment]
On 8a5cc54: 

```bash
$ tox -- -k 'loggly'
# ...
FAILED tests/test_logs.py::test_loggly_log[R_tp_opt] - NameError: name 'self' is not defined
FAILED tests/test_logs.py::test_loggly_log[R_bare] - NameError: name 'self' is not defined
======================================== 2 failed, 2 passed, 46 deselected, 4 warnings in 0.83s ========================================
```

**Comment by ethho** [comment]
Coverage in the following pytest: https://github.com/TACC-Cloud/python-reactors/blob/135191ba4c4195d76ab0d170da3923f51a786109/tests/test_logs.py#L132-L158
